mqtt-impl/src/main/java/io/streamnative/pulsar/handlers/mqtt/MQTTServerConfiguration.java

```diff
@@ -97,6 +97,14 @@
     )
     private String defaultNamespace = "default";
 
+
+    @FieldContext(
+            category = CATEGORY_MQTT,
+            required = true,
+            doc = "Default Pulsar topic that the MQTT server used."
```

Comment:
Default Pulsar topic that the MQTT server used. -> Default Pulsar topic domain the MQTT server used.